src/common/components/traffic-lights/index.js

```diff
@@ -45,13 +45,13 @@ export default class TrafficLights extends Component {
 
     this.state = {
       signals: [{
-        label: '1',
+        label: '',
```

Comment:
If we don't use the label, let's make it optional (in Signal component), so we don't have to specify it here at all.